pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -44,12 +44,15 @@ const (
 
 // HttpForwarderHandlerV2 is a PipelineHandler which sends metrics to another gostatsd instance
 type HttpForwarderHandlerV2 struct {
-	postId             uint64 // atomic - used for an id in logs
-	messagesInvalid    uint64 // atomic - messages which failed to be created
-	messagesCreated    uint64 // atomic - messages which were created
-	messagesSent       uint64 // atomic - messages successfully sent
-	messagesRetried    uint64 // atomic - retries (first send is not a retry, final failure is not a retry)
-	messagesDropped    uint64 // atomic - final failure
+	postId           uint64       // atomic - used for an id in logs
+	messagesInvalid  uint64       // atomic - messages which failed to be created
+	messagesCreated  uint64       // atomic - messages which were created
+	messagesSent     uint64       // atomic - messages successfully sent
+	messagesRetried  uint64       // atomic - retries (first send is not a retry, final failure is not a retry)
+	messagesDropped  uint64       // atomic - final failure
+	postLatencyTotal int64        // atomic - total of the time taken to send messages in a flush interval
```

Comment:
We should make this `atomic.Int64` as well.  This code pre-dates the `atomic` package, hence why we don't have it.